crates/aptos/src/move_tool/bytecode.rs

```diff
@@ -330,15 +342,40 @@ impl BytecodeCommand {
         &self,
         bytecode_path: &Path,
         version: DisassemblerVersion,
+        print_code_size: Option<Vec<String>>,
     ) -> Result<String, CliError> {
         match version {
-            DisassemblerVersion::V1 => self.disassemble_v1(bytecode_path),
-            DisassemblerVersion::V2 => self.disassemble_v2(bytecode_path),
+            DisassemblerVersion::V1 => self.disassemble_v1(bytecode_path, print_code_size),
+            DisassemblerVersion::V2 => self.disassemble_v2(bytecode_path, print_code_size),
         }
     }
 
-    fn disassemble_v1(&self, bytecode_path: &Path) -> Result<String, CliError> {
+    fn disassemble_v1(
+        &self,
+        bytecode_path: &Path,
+        print_code_size: Option<Vec<String>>,
+    ) -> Result<String, CliError> {
         let bytecode_bytes = read_from_file(bytecode_path)?;
+        let print_code_bool = print_code_size.is_some_and(|v| {
```

Comment:
This seems common between v1 and v2 flows?

Reply:
done